local-kubernetes/README.md

```diff
@@ -0,0 +1,56 @@
+# Local Kubernetes Development Environment
+
+This folder contains the scripts necessary to run the entire environment locally. 
+
+## Set Up Kubernetes
+
+To begin, make sure you have kubernetes running locally as per the [Kubernetes Dev Setup Instructions](../Docs/DevSetup/local-k8s-setup.md)
+
+Create the buying catalogue namespace - `kubectl apply -f local-namespace.yml`
+
+To run the system, you need to create a secret in kubernetes to access the private container registry, as per the [Connect Local Kubernetes with our Private Container Registry Instructions](../Docs/DevSetup/k8s-private-registry.md).
+
+You will also need a secret for the local instances of sql server - `kubectl create secret generic mssql --from-literal=SA_PASSWORD="<YOUR PASSWORD>" -n buyingcatalogue`. **Password must be 8 characters**.
+
+## Launching / Tearing Down the Environment
+
+Then, run `launch-environment{.sh|ps1}` to launch the environment.
+The script will start, available on localhost:
+- Document Service
+  - [DAPI](http://localhost:5101/swagger) on port 5101 
+  - Azurite on ports 10000 (blob),10001 (queue),10002 (table), with development user default (access with [Azure Storage Explorer](https://azure.microsoft.com/en-gb/features/storage-explorer/))
+- Buying Catalogue Service
+  - [BAPI](http://localhost:5100/swagger) on port
+  - SQL Server on port 1450 (log in localhost,1450 in [SSMS](https://docs.microsoft.com/en-us/sql/ssms/download-sql-server-management-studio-ssms?view=sql-server-ver15))
+- [Marketing Pages](http://localhost:3002/supplier/solution/100000-001/preview) on port 3002
+- [Public Browse](http://localhost:3000/) on port 3000
+
+Run `tear-down-environment{.sh|ps1}` to tear down the environment.
+
```

Comment:
is it worth having a section for how to connect Azure storage explorer to the local azurite container, and an SQL client to the local DB?